src/components/Person/tabs/details/I18nForm.js

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import classNames from 'classnames';
+import { cloneDeep } from 'lodash';
```

Comment:
Are you sure this way we're not importing all of lodash ?
I'm concerned with package size. Shouldn't it be something like `lodash/cloneDeep` ?

Reply:
you right